public/js/app.js

```diff
@@ -23,10 +23,10 @@ projectMigrationApp.run(function($rootScope) {
     };
   } else {
     $rootScope.urls = {
-      'projectsUrl': '/projects.json',
-      'migrationsUrl': '/migrations',
-      'migratedUrl': '/migrated',
-      'projectUrl': '/projects/'
+      'projectsUrl': '/ctools-project-migration/projects.json',
+      'migrationsUrl': '/ctools-project-migration/migrations',
+      'migratedUrl': '/ctools-project-migration/migrated',
+      'projectUrl': '/ctools-project-migration/projects/'
```

Comment:
Good point. This is actually related to another request down the road. I added your request as a TODO comment inside the code.